ci-operator/step-registry/azure/validate/bastionhost/azure-validate-bastionhost-commands.sh

```
@@ -0,0 +1,218 @@
+    local host=$3
+    local remote_cmd=$4
+    local description=$5
+
+    log "Running: ${description}"
+    options="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ServerAliveInterval=30 -o ServerAliveCountMax=5 -o ConnectTimeout=10"
+
+    # Disable tracing for SSH commands to avoid leaking sensitive data
+    [[ $- == *x* ]] && WAS_TRACING=true || WAS_TRACING=false
+    set +x
+
+    output=$(ssh ${options} -i "${sshkey}" ${user}@${host} "${remote_cmd}" 2>&1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)